Fixed #34210 -- Added unittest's durations option to the test runner. (#16976)
django/django: docs/releases/5.0.txt

Tests
~~~~~

> * :class:`~django.test.runner.DiscoverRunner` and the :djadmin:`test` 
>   management can now track the duration of slow test cases. This can be enabled 
>   by using the :option:`test --durations` option and requires Python 3.12+.

Reviewer: ```suggestion
* The :djadmin:`test` command and the
  :class:`~django.test.runner.DiscoverRunner` class can now show the durations
  of the slowest tests on Python 3.12+. Enable with the
  :option:`test --durations` option.
```

  :meth:`~django.test.Client.asession`, :meth:`~django.test.Client.alogin`,
  :meth:`~django.test.Client.aforce_login`, and
  :meth:`~django.test.Client.alogout`.
* :class:`~django.test.runner.DiscoverRunner` and the :djadmin:`test` 
  management can now track the duration of slow test cases. This can be enabled 
  by using the :option:`test --durations` option and requires Python 3.12+.
* The :djadmin:`test` command and the

>   :class:`~django.test.runner.DiscoverRunner` class can now show the durations

Reviewer 2: ```suggestion
  :class:`~django.test.runner.DiscoverRunner` class can now show the duration
```
The Sphinx spell checker seems not happy with `durations` here
